packages/components/src/notice/README.md

```diff
@@ -42,43 +30,45 @@ Notices are color-coded to indicate the type of message being communicated:
 
 If an icon is included in the Notice, it should be color-coded to match the Notice state.
 
+### Do's and Don'ts
```

Comment:
I tried to reorganize this to be more clear by addressing a few things:

1. Moved the text above the images and added a separator between them to make it clear which caption belonged to which image
2. Removed the repetitive Do/Don't at the start of each statement. I think it reads better this way

These are obviously my own preferences, so I'm happy to be overruled. I kept them in separate commits, so they're really simple to ditch!